index.html

```diff
@@ -1,52 +1,352 @@
+                                </div>
+                            </div>
+                            <div class="card-see">
+                                <div class="see-project">
+                                    <a href="http://#"> See Project</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="card-headerbg">
+                    </div>
+                    <div class="card">
+                        <div class="card-gra">
+                            <div class="card-header">
+                                <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            </div>
+                            <div class="card-body">
+                                <div class="language card-ruby">
+                                    <a href="http://">Ruby on Rails</a>
+                                </div>
+                                <div class="language card-css">
+                                    <a href="http://">Css</a>
+                                </div>
+                                <div class="language card-js">
+                                    <a href="http://">JavaScript</a>
+                                </div>
+                                <div class="language card-html">
+                                    <a href="http://">HTML</a>
+                                </div>
+                            </div>
+                            <div class="card-see">
+                                <div class="see-project">
+                                    <a href="http://#"> See Project</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+
+        <section class="about">
+
+
+            <div class="about-header">
+                <h1>About me</h1>
+            </div>
+
+            <div class="about-content">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.
+                </p>
+            </div>
+
+            <div class="resume">
+                <div class="see-project">
+                    <a href="http://#" id="about-resume">Get my Resume</a>
+                </div>
+            </div>
+
+            <div class="box-tech">
+
+                <div class="tech">
+                    <div class="box">
+                        <div class="box-icon"><img src="assets/img/language.svg" alt="" srcset=""></div>
+                        <div class="box-descr">
+                            <h1>Languages</h1>
```

Comment:
- [ ] kindly consider adding the technology categories ("Languages", "Frameworks" and "Skills")  in a `ul`, each of them in a `li` because they are used to represent an item in a list and they are